Android Build Failed

Hi, 

In my  Cordova  project, I am using analytics  plugin version 11.6.2, however, the build gets failed with the below error 

[13:26:51] FAILURE: Build failed with an exception.
[13:26:51] * What went wrong:
[13:26:51] Execution failed for task ':app:processReleaseManifest'.
[13:26:51] > Manifest merger failed : Attribute receiver#com.google.android.gms.analytics.AnalyticsReceiver@exported value=(true) from AndroidManifest.xml:28:42-65
[13:26:51]   	is also present at [com.google.android.gms:play-services-analytics:11.6.2] AndroidManifest.xml:27:108-132 value=(false).

--- a/plugin.xml
+++ b/plugin.xml
@@ -65,7 +65,7 @@
     </config-file>
     <!-- Add support for devices without Google Play Services installed. -->
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
-      <receiver android:name="com.google.android.gms.analytics.AnalyticsReceiver" android:enabled="true">
+      <receiver android:name="com.google.android.gms.analytics.AnalyticsReceiver" android:enabled="true" android:exported="true">
         <intent-filter>
           <action android:name="com.google.android.gms.analytics.ANALYTICS_DISPATCH" />
         </intent-filter>

